specification/reservations/resource-manager/Microsoft.Capacity/preview/2019-04-01/examples/PurchaseReservationOrder.json

```diff
@@ -11,6 +11,7 @@
         "reservedResourceType": "VirtualMachines",
         "billingScopeId": "/subscriptions/ed3a1871-612d-abcd-a849-c2542a68be83",
         "term": "P1Y",
+        "billingPlan": "Monthly",
```

Comment:
>"billingPlan": "Monthly", [](start = 8, length = 25)

What happens if the request does not specify `billingPlan`?

Reply:
The billing plan will be defaulted to "Upfront" which was standard behavior before this update.

Reply:
Great. Thanks for confirming.